script.js

```diff
@@ -0,0 +1,21 @@
+var square = 49;
+
+$(document).ready(function () {
+    var gameboard = $("#gameboard");
+});
+
+var letters = ['', 'A', 'B', 'C', 'D', 'E', 'F'];
+var numbers = [0, 1, 2, 3, 4, 5, 6];
+
+var content = ""
+
+for (var i = 0; i < square; i++) {
```

Comment:
It would be beter to use two parameters (`rows` / `cols`) with two loops to define game board size.
```
   var colsCount = 7;
   var rowsCount = 7;
   for(var x = 0; x < colsCount; x++) {
      for(var y = 0; y < rowsCount; y++) {
      ...
```